plane/source/docs/quadplane-vtol-tuning-process.rst

```diff
@@ -0,0 +1,276 @@
+
+Test QHOVER
+-----------
+
+This test will allow to test the altitude controller and ensure the stability of your aircraft.
+
+1. Check :ref:`Q_M_HOVER_LEARN <Q_M_HOVER_LEARN>` is set to 2. This will allow the controller to learn by itself the correct hover value when flying.
+
+2. Take off in QSTABILIZE and increase altitude to 5m. Switch to QHOVER and be ready to switch back to QSTABILIZE. If the aircraft is hovering at a very low hover throttle value you may hear a reasonably fast oscillation in the motors. Ensure the aircraft has spent at least 30 seconds in hover to let the hover throttle parameter converge to the correct value. Land and disarm the aircraft.
+
+3. Set these parameters on ground and preferably disarm  (A confident pilot could set them in flight with GCS):
+
+  - :ref:`Q_P_ACCZ_I <Q_P_ACCZ_I>` to 2 x :ref:`Q_M_THST_HOVER <Q_M_THST_HOVER>`
+  - :ref:`Q_P_ACCZ_P <Q_P_ACCZ_P>` to :ref:`Q_M_THST_HOVER <Q_M_THST_HOVER>`
+
+ If the QuadPlane in QHOVER starts to move up and down, the vertical position and velocity controllers may need to be reduced by 50%. These values are: :ref:`Q_P_POSZ_P <Q_P_POSZ_P>` and :ref:`Q_P_VELZ_P <Q_P_POSZ_P>`.
+
+Evaluating the aircraft tune
+----------------------------
+
+Most pilots will look to move to :ref:`QAUTOTUNE<qautotune-mode>` as quickly as possible once their aircraft can hover safely in QHOVER. Before QAUTUTUNE mode is run, the pilot should ensure that the current tune is good enough to recover from the repeated tests run in QAUTOTUNE mode. To test the current state of tune:
+
+1. Take off in QHOVER or QSTABILIZE
+2. Apply small roll and pitch inputs. Start with 5 degree inputs and releasing the stick to centre, pitch, left, right, roll forward back, then all 4 points on the diagonal
+3. Increase inputs gradually to full stick deflection
+4. Go to full stick deflection, quickly momentarily, and let the sticks spring back to centre
+
+If the aircraft begins to overshoot significantly or oscillate after the stick input, halt the tests before the situation begins to endanger the aircraft. The aircraft may require manual tuning described below before QAUTOTUNE can be run.
+
+To test the stabilization loops independent of the input shaping, set the parameter: :ref:`Q_A_RATE_FF_ENAB <Q_A_RATE_FF_ENAB>` to 0.
+
+1. Take off in QHOVER or QSTABILIZE
+2. Hold a roll or pitch input
+3. Release the stick and observe the overshoot as the aircraft levels itself
+4. Gradually increase the stick deflection to 100%
+
+Halt the tests if the aircraft overshoots level significantly or if the aircraft oscillates, the aircraft may require manual tuning (:ref:`see next section <ac_rollpitchtuning>`) before QAUTOTUNE can be run.
+
+Set :ref:`Q_A_RATE_FF_ENAB <Q_A_RATE_FF_ENAB>` to 1 after the tests are complete.
+
+Manual tuning of Roll and Pitch
```

Comment:
I think filtering should be done before D gains

Reply:
done